manifests/kerberos.pp

```diff
@@ -19,7 +19,7 @@
   }
 
   if ! empty($_kerberos_login_principals) {
-    $k5login = ['# File managed by Puppet (root::manage_kerberos = true), DO NOT EDIT'] + $_kerberos_login_principals
+    $k5login = ['# File managed by Puppet (root::manage_kerberos = true), DO NOT EDIT'] + $_kerberos_login_principals + ["\n"]
```

Comment:
Because this is using `join` with `"\n"` I believe you'll end up with 2 new lines at end of file.  So when I run into this I just sensure there is empty string at end
```suggestion
    $k5login = ['# File managed by Puppet (root::manage_kerberos = true), DO NOT EDIT'] + $_kerberos_login_principals + ['']
```